language-support/qute/qute.tmLanguage.json

```diff
@@ -77,7 +77,7 @@
 			"name": "string.unquoted.cdata.qute"
 		},
     "section_start_tag": {
-      "begin": "({)(#)(else\\sif|\\w+(\\.\\w+)*)",
+      "begin": "({)(#)(else\\sif|(\\w+(\\.\\w+)*)^(?!.*(return|break|case|continue|default|do|while|for|switch|if|else|include)))?",
```

Comment:
I think it's not the correct fix,because 

 * `{include}` should be not higligthed as keyword.
 * `{#include`  must be higlighted as keyword.

perhaps a correct fix is to have `section_start_tag` and `section_core_start_tag` (which higlight with,include,etc)